src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,99 @@
+package storage;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+import tasklist.TaskList;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.ArrayList;
+
+public class Storage {
+    private String filepath;
+    private Path path;
+    public Storage(String filepath) {
+        this.filepath = filepath;
+        this.path = Paths.get(this.filepath);
+    }
+    /**
+     * Create the directory and file to store the previous list, else update it to the current list.
```

Comment:
Perhaps the start of javadoc should be a verb with s behind according to the style guide?
eg, Creates instead of Create